src/js/post-911-gib-status/containers/StatusPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { Link } from 'react-router';
+
+import FormTitle from '../../common/schemaform/FormTitle';
+
+import { getEnrollmentData } from '../actions/post-911-gib-status';
+import EnrollmentHistory from '../components/EnrollmentHistory';
+import UserInfoSection from '../components/UserInfoSection';
+
+class StatusPage extends React.Component {
+  componentDidMount() {
+    this.props.getEnrollmentData();
+  }
+
+  render() {
+    // TODO: change the service name below from "user-profile" to
```

Comment:
This TODO got pasted in here by mistake. Related, I filed https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3329 for us to update that service name